index.html

```diff
@@ -138,6 +138,60 @@ <h4 class="medium-description">CANESTRELLI / $18</h4>
           </div>
         </div>
       </section>
+      <section class="sweets-foods-wrapper">
+        <div class="inner-wrapper">
+          <div class="sweets-foods-boxes-wrapper">
+            <div class="sweets-foods-box">
+              <img src="assets/images/pexels-marta-dzedyshko-2067396.jpg" />
+            </div>
+            <div class="sweets-foods-box">
+              <h4 class="medium-description">History</h4>
+              <h3 class="main-small-heading">
+                <span>Strange story of the</span
+                ><span>emergence of healthy food</span>
+              </h3>
+              <p class="main-description">
+                <span>Sample text. Click to select the text box.</span>
+                <span>Click again or double click to start editing</span>
+                <span>the text.</span>
+              </p>
+              <h5 class="small-desrciption">Read More</h5>
+            </div>
+            <div class="sweets-foods-box">
+              <img src="assets/images/pexels-photo-4099237.jpeg" />
+            </div>
+            <div class="sweets-foods-box">
+              <h4 class="medium-description">Comiunity</h4>
+              <h3 class="main-small-heading">
+                <span>Healthy food in social</span><span>networks, with us</span
+              ><span>interesting</span>
+              </h3>
+              <p class="main-description">
+                <span>Sample text. Click to select the text box.</span>
+                <span>Click again or double click to start editing</span>
+                <span>the text.</span>
+              </p>
+              <h5 class="small-desrciption">Read More</h5>
+            </div>
+            <div class="sweets-foods-box">
+              <img src="assets/images/pexels-photo-949069.jpeg" />
+            </div>
+            <div class="sweets-foods-box">
+              <h4 class="medium-description">Philosophy</h4>
+              <h3 class="main-small-heading">
+                <span>Only the best food, the best</span
+                ><span>musicians the best guests</span>
+              </h3>
+              <p class="main-description">
+                <span>Sample text. Click to select the text box.</span>
+                <span>Click again or double click to start editing</span>
+                <span>the text.</span>
+              </p>
+              <h5 class="small-desrciption">Read More</h5>
```

Comment:
Takie elementy zazwyczaj są <a>.
Jeśli nie zna się linku, to jako wartość atrybutu href podaje się "#"

Reply:
Poprawione